packages/service/src/utils/users.test.ts

```diff
@@ -0,0 +1,32 @@
+import { pool, db } from "./../db";
+import { createUser, selectUsers, validateUser } from "./users";
+
+afterAll(() => {
+  return pool.end();
+});
+
+test("creating a user", async () => {
+  const user = await createUser({ name: "Toby Savage 2", password: "12345" });
+  console.log(user);
+  expect(user).toBeDefined();
+});
+
+// Toby Savage 2's ID: "3008c2a6-d570-4434-b676-7b345b7afae7"
+test("validate a user", async () => {
+  const user = (
+    await validateUser({
+      name: "Toby Savage 2",
+      password: "12345",
+      id: "3008c2a6-d570-4434-b676-7b345b7afae7",
```

Comment:
Hardcoding ids from your local database is not thing thing to do

Reply:
https://jestjs.io/docs/setup-teardown#scoping

Maybe we can use describe blocks to reuse the same fixture in both tests